tools/src/test/resources/1.5/valid-saasbom-1.5.json

```diff
@@ -0,0 +1,294 @@
+    "component": {
+      "bom-ref": "acme-stock-application",
+      "type": "application",
+      "name": "Acme SaaSBOM Example",
+      "version": "2022-1"
+    }
+  },
+  "services": [
+    {
+      "bom-ref": "stock-ticker-service",
+      "provider": {
+        "name": "Acme Inc",
+        "url": [ "https://example.com" ]
+      },
+      "group": "com.example",
+      "name": "Stock Ticker Service",
+      "version": "2022-1",
+      "endpoints": [
+        "https://example.com/",
+        "https://example.com/app"
+      ],
+      "authenticated": true,
+      "trustZone": "Acme Public Zone",
+      "data": [
+        {
+          "name": "Consumer to Stock Service",
+          "description": "Traffic to/from consumer to service",
+          "classification": "Customer",
+          "flow": "bi-directional",
+          "source": [
+            "https://0.0.0.0"
+          ],
+          "destination": [
+            "https://0.0.0.0"
+          ]
+        },
+        {
+          "name": "Stock Service to MS-1",
+          "description": "Traffic to/from stock service to microservice-1",
+          "classification": "PII",
+          "flow": "bi-directional",
+          "source": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-1.example.com"
+          ],
+          "destination": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-1.example.com"
+          ]
+        },
+        {
+          "name": "Stock Service to MS-2",
+          "description": "Traffic to/from stock service to microservice-2",
+          "classification": "PIFI",
+          "flow": "bi-directional",
+          "source": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-2.example.com"
+          ],
+          "destination": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-2.example.com"
+          ]
+        },
+        {
+          "name": "Stock Service to MS-3",
+          "description": "Traffic to/from stock service to microservice-3",
+          "classification": "Public",
+          "flow": "bi-directional",
+          "source": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-3.example.com"
+          ],
+          "destination": [
+            "urn:cdx:3e671687-395b-41f5-a30f-a58921a69b79/1#ms-3.example.com"
+          ]
+        }
+      ],
+      "externalReferences": [
+        {
+          "type": "documentation",
+          "url": "https://example.com/app/swagger"
+        }
+      ],
+      "services": [
+        {
+          "bom-ref": "ms-1.example.com",
+          "provider": {
+            "name": "Acme Inc",
+            "url": [ "https://example.com" ]
+          },
+          "group": "com.example",
+          "name": "Microservice 1",
+          "version": "2022-1",
+          "description": "Example Microservice",
+          "endpoints": [
+            "https://ms-1.example.com"
+          ],
+          "authenticated": true,
```

Comment:
I wonder if this might be a place where the https://identity.foundation/presentation-exchange/ spec could be used to specify the requirements around authentication. We could imagine a world where scorecard probes are hosted as microservices executed as an IPVM affect.

Reply:
Just spitballin here.

Reply:
I like your thinking @pdxjohnny. 😄 We've wanted to be able to document authentication requirements in CycloneDX for a while now, but as you know, it's not a simple task. We haven't had an opportunity to research existing specs yet, but the presentation exchange format looks interesting and may just work. Historically, we've supported links out to other schemas/files via CycloneDX [external references](https://cyclonedx.org/docs/1.4/json/#services_items_externalReferences). We're doubling the number of external references supported in the upcoming v1.5 release. Most of the new external references for v1.5 are [here](https://github.com/CycloneDX/specification/pull/189/files). Integration could be as simple as adding a new one for authentication requirements, or we may choose a more integrated approach. Regardless, I'd recommend creating a new ticket to discuss this, as this ticket is specific to dataflow enhancements.

Reply:
Sounds great :D will do! Thank you for your response